Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -140,8 +140,11 @@ struct CombinatorialKalmanFilterResult {
   // Fitted states that the actor has handled.
   MultiTrajectory<source_link_t> fittedStates;
 
-  // The indices of the 'tip' of the tracks stored in multitrajectory.
-  std::vector<size_t> trackTips;
+  // This is the indices of the 'tip' of the tracks stored in multitrajectory.
+  std::vector<size_t> lastMeasurementIndices;
+
+  // This is the indices of the 'tip' of the tracks stored in multitrajectory.
```

Comment:
This is the same comment above the two, can you specify the difference, please?

Reply:
My bad, I pushed the updated comments. One refer to the Indices of the last state of the multitrajectory while the other refer to the last measurement state.

Reply:
Excellent.